src/main/java/org/scijava/log/DefaultLogger.java

```diff
@@ -64,7 +64,7 @@ public void setLogLevel(final String classOrPackageName, final LogLevel level) {
 
 	@Override
 	public void alwaysLog(final LogLevel level, final Object msg, final Throwable t) {
-		notifyListeners(level, msg, t);
+		notifyListeners(this, level, msg, t);
```

Comment:
Cool, I think this is a good idea. But we need to maintain backwards compatibility. The old `notifyListeners` method should remain (possibly/probably deprecated), with its source being the default channel.